notebooks/evaluation/evaluation_utilities.py

```
@@ -0,0 +1,340 @@
+from ragas.metrics._domain_specific_rubrics import RubricsScore
```

Ruff (0.11.9)

`ragas.metrics._domain_specific_rubrics.RubricsScore` imported but unused

Remove unused import: `ragas.metrics._domain_specific_rubrics.RubricsScore`

(F401)